lib/transition.js

```diff
@@ -294,6 +308,24 @@ class Transition {
       }
       return undefined
     }
+
+    /**
+     * Check if transition prop is inside vendor specific rule
+     */
+    inVendorSpec (decl) {
+      let { parent } = decl
+      let { selector, type } = parent
```

Comment:
I prefer to have explicit `parent.type`. It is easy to misunderstand code above by thinking that `type` is `decl.type`.

Reply:
Agree with you, it could be misleading. Rewrote with ```parent.type``` and ```parent.selector```.